### ISSUE TITLE
Path to vulkan headers

### ISSUE THREAD

**Issue by xelibrion** [mpv-player/mpv#12673]
I'm trying to build libmpv on my M2 Mac with support for Vulkan
```
meson setup build
meson configure build -Dlibmpv=true -Dvulkan=enabled -Dvulkan-interop=enabled -Dios-gl=disabled
meson compile -C build
```

but my build is failing with 
```
/Users/xelibrion/homebrew/Cellar/ffmpeg/HEAD-7218574_1/include/libavutil/hwcontext_vulkan.h:25:10: fatal error: 'vulkan/vulkan.h' file not found
```

I have `vulkan-loader` and `vulkan-headers` installed via homebrew. As far as I understand, `vulkan-loader` comes with `.pc` file for `package-config`, but `vulkan-headers` does not. 

I can see in a call to `cc`, there's param `-I/Users/xelibrion/homebrew/Cellar/vulkan-loader/1.3.268/include`, which points to `vulkan-loader` folder, but not `vulkan-headers`. What is the most conventional way of including headers for vulkan to compile?

Full build log here: https://pastebin.com/evW0z8eV

**Comment by xelibrion** [comment]
> That's strange. `.pc` file from `vulkan-loader` should cover both loader and headers. 1.3.267 has a regression in the `vulkan.pc` but it was fixed in 1.3.268 ([KhronosGroup/Vulkan-Loader#1320](https://github.com/KhronosGroup/Vulkan-Loader/pull/1320)).
> 
> Can you try building with [this guide](https://github.com/m154k1/mpv-build-macOS)?

Commented as https://github.com/m154k1/mpv-build-macOS/issues/1

**Comment by Dudemanguy** [comment]
Looks like the problem was solved over there?
